tests/unit/AgmLocation.spec.ts

```diff
@@ -89,8 +89,7 @@ describe('AgmLocation', () => {
       })
 
     const input = wrapper.find('#agm-location')
-    const a = 'a'
-    const a101 = a.repeat(101)
+    const a101 = 'a'.repeat(101)
     await input.setValue(a101)
```

Comment:
Don't change it this time, but in the future you may want to simulate a user typing in a bunch of "a"s so that it exercises the debouncing/validation code as realistically as possible.

Reply:
Got it, I'll keep that mind.